plotting.py

```diff
@@ -0,0 +1,211 @@
+    sns.regplot(x='lidar_max', y='insitu_max',
+                data=all_heights_gdf, color="purple", ax=ax1)
+    sns.regplot(x='lidar_mean', y='insitu_mean',
+                data=all_heights_gdf, color="purple", ax=ax2)
+
+    # Put a title together.
+    title1 = "Tree Heights: Lidar Calculated vs. Manual Measurement\n"
+    title2 = "{} Study Plot\n".format(study_site.upper())
+    title3 = "Max Height, Mean Height"
+    title = title1 + title2 + title3
+
+    fig.suptitle(title, size='medium')
+
+    plt.show()
+###################################
+
+
+def plot_maps(
+        lidar, plots, study_site,
+        cntry_path, states_path,
+        roads_path, aoi_path):
+    """ 
+    Creates a quick plot given a raster image and point file.
+
+    Parameters:
+    ----------
+    lidar: datarray
+        A datarray - Landsat in this case.
+    plots: geodataframe
+        A geodatframe containing point locations
+    study_site: str
+        Name of study site.
+    cntry_path: str
+        Path to country boundary file.
+    states_path: str
+        Path to states boundary files.
+    roads_path: str
+        Path to road geometry file.
+    aoi_path: str
+        Path to study site boundary geometry.
+
+    Returns:
+    -------
+    A plot to screen.
+
+    """
+
+    # Create map plots showing sjer location.
+    usa_gdf = gpd.read_file(cntry_path)
+    states_gdf = gpd.read_file(states_path)
+    aoi_gdf = gpd.read_file(aoi_path)
+    roads_gdf = gpd.read_file(roads_path)
+
+    # Reproject to common crs.
+    aoi_reproj_gdf = aoi_gdf.to_crs(usa_gdf.crs)
+    roads_reproj_gdf = roads_gdf.to_crs(plots.crs)
+
+    # Clip down roads to only lidar extent.
+    roads_reproj_clip_gdf = gpd.clip(roads_reproj_gdf, lidar.rio.bounds())
+
+    # Create a point representing the area of interest.
+    # Easier for plotting.
+    aoi_point = aoi_reproj_gdf["geometry"].centroid
+
+    # Create larger scale map.
+    fig1, (ax1) = plt.subplots(1, 1, figsize=(10, 6))
+
+    # zoom in to map using.
+    ax1.set(xlim=[-125, -110], ylim=[30, 45])
+    ax1.yaxis.set_major_locator(plt.MaxNLocator(3))
+    ax1.set_xlabel('')
+    ax1.set_ylabel('')
+
+    usa_gdf.plot(ax=ax1)
+    states_gdf.plot(ax=ax1)
+    aoi_point.plot(ax=ax1, color='r')
+
+    ax1.set_title("Location of NEON {} Field Site".format(study_site.upper()))
+
+    # Create Lidar map with roads.
+    fig2, (ax2) = plt.subplots(1, 1, figsize=(10, 6))
+
+    lidar.plot(ax=ax2, label="CHM (m)")
+    plots.plot(ax=ax2, marker='s', markersize=50000, color='red')
+    roads_reproj_clip_gdf.plot(
+        ax=ax2, color='black', linewidth=2, label='Roads')
+    # Addding a legend causes the cell execution to take a long time.
+    # I dont know why this is happening.
+    # ax2.legend()
+
+    # Capitalize study_site for title.
+    title = "LIDAR CHM and {} Study Sites".format(study_site.upper())
+    ax2.set_title(title)
+    ax2.set_xlabel('')
+    ax2.set_ylabel('')
+
+    plt.show()
```

Comment:
Nice addition - I like this figure of the study sites. As a suggestion, you could differentiate the 2 sites with different markers; also add a label to the color bar

Reply:
I had some problems with this.  For some reason I couldnt control the size of the study site markers.  I gave up trying and forgot about it.  And you're right, I forget label the colorbar.